### PR TITLE
XFA - Don't display print-only elements

### PR DESCRIPTION
**Pull request by calixteman**
- partial fix for https://bugzilla.mozilla.org/show_bug.cgi?id=1716980.

### CODE REVIEW

File: web/xfa_layer_builder.css [hunk 1]
```diff
@@ -13,6 +13,10 @@
  * limitations under the License.
  */
 
+.xfaPrintOnly {
+  display: none;
```
Comment (marco-c) [inline web/xfa_layer_builder.css:17]: Shouldn't these elements be displayed when the user wants to print the PDF?

Comment (calixteman) [reply web/xfa_layer_builder.css:17]: I need to check if it's relevant or not in general. For example in the pdf in the above bug, I'm not sure if it's correct correct to print these elements when maybe they should be dynamically modified.
So for now we just print what the user see.

Comment (marco-c) [reply web/xfa_layer_builder.css:17]: OK, can you file an issue about this so we don't forget?

**Review by marco-c — APPROVED** [review]

**Pull request merged**